app/controllers/profiles_controller.rb

```diff
@@ -45,7 +45,7 @@ def dashboard
     @passengers = Passenger.where(journey_id: Journey.where(user: @user))
 
 
-    else
+    elsif @user.cars[0]
```

Comment:
That would be more efficient only if that was the only thing you checked (existence). But later you do `@car = ...`

Use `.first` in this case.